### PR TITLE
Optimised Movement Code

### PR DESCRIPTION
**Pull request by abhinav3967**
Made some variables constants ,used terenary if's and removed some white spaces

### CODE REVIEW

File: Characters/BaseFighter/BaseFighter.gd [hunk 1]
```diff
@@ -48,17 +51,12 @@ func _ready():
...
 	velocity.y = jump_velocity
 
 func _check_is_grounded():
 	for raycast in ground_raycasts.get_children():
-		if raycast.is_colliding():
-			return true
-	
-	
-	return false
+			return raycast.is_colliding()
```
Comment (Technohacker) [inline Characters/BaseFighter/BaseFighter.gd:59]: Quite a lot of indentation here :P

File: project.godot [hunk 2]
```diff
@@ -8,14 +8,9 @@
 
 config_version=4
 
-_global_script_classes=[ {
-"base": "Node",
-"class": "StateMachine",
-"language": "GDScript",
-"path": "res://StateMachine.gd"
-} ]
+_global_script_classes=[  ]
```
Comment (Technohacker) [inline project.godot:11]: What happened to the state machine?

**Review by Technohacker — CHANGES_REQUESTED** [review]
Few changes needed :P

File: project.godot (hunk 2 above)
Comment (abhinav3967) [reply project.godot:11]: State Machine not implemented yet

File: Characters/BaseFighter/BaseFighter.gd (hunk 1 above)
Comment (abhinav3967) [reply Characters/BaseFighter/BaseFighter.gd]: fixed it

**Review by Technohacker — APPROVED** [review]

**Pull request merged**